tests/integration/stop_condition_nested_state/resources/scl.txt

```diff
@@ -0,0 +1,3 @@
+eventually actuator.nestED.param > 0;
```

Comment:
Small thing.... Nested seems to start with param = 1....  Then increases param on tick. Could we do > 1 here?